component.json

```diff
@@ -11,6 +11,9 @@
           "label": "JSONata Expression",
           "required": true,
           "viewClass": "JSONataView"
+        },
+        "manualTransform": {
```

Comment:
Can we change this to extendedFunctions? Right now the terminology is targeted toward the implementation.